apps/parse/lib/parse/vehicle_positions.ex

```diff
@@ -1,38 +1,54 @@
 defmodule Parse.VehiclePositions do
   @moduledoc """
 
-  Parser for the VehiclePositions.pb GTFS-RT file.
+  Parses an enhanced Vehicle Position JSON file into a list of `%Model.Vehicle{}` structs.
 
   """
   @behaviour Parse
   alias Model.Vehicle
-  alias Parse.Realtime.FeedMessage
   import Parse.Helpers
 
-  def parse(blob) do
-    blob
-    |> FeedMessage.decode()
-    |> (fn message -> message.entity end).()
-    |> Stream.map(fn entity -> entity.vehicle end)
-    |> Stream.map(&parse_vehicle_update/1)
+  @impl true
+  def parse(body) do
+    body
+    |> Jason.decode!()
```

Comment:
If you give Jason the `strings: :copy` option (see https://hexdocs.pm/jason/Jason.html#decode/2-options), then you don't need to use `optional_field_copy` elsewhere in the module.